py/configu/utils.py

```diff
@@ -25,13 +50,21 @@ def parse_template(template: str) -> List[str]:
     template_vars = []
     for tag_type, tag_key in chevron.tokenizer.tokenize(template):
         if tag_type not in ["variable", "literal"]:
-            raise Exception(
-                error_message(f'invalid template "{template}"', ["Template", "parse"])
-            )
+            # Why is this not a ConfigError?
+            raise Exception(f"template '{template}' mustn't contain unsupported tokens")
```

Comment:
@rannn505 any good reason why this time it's plain error instead of `ConfigError`?

Reply:
@RonConfigu @jamesL92 Not sure really, I guess this one is wrapped in an upper scope with a peoper ConfigError like here
![image](https://github.com/configu/configu/assets/11806915/065b2326-4aa2-4bd0-ab14-7a936a82ec21)

Reply:
May be worth raising it as a `ConfigError` in that case, as it'd be the only exception raised in the entire python SDK that isn't a `ConfigError`. I had a look through and there is a path via which this could be raised and not be wrapped in a `ConfigError` upstream: 
`EvalCommand.run` -> `EvalCommand._eval_templates` -> `utils.parse_template`.

Also it may be worth changing the docstrings [here](https://github.com/configu/configu/blob/329-update-and-align-py-ts/py/configu/commands/upsert_command.py#L58) and [here](https://github.com/configu/configu/blob/3790cf84feb7409d08d685d41bda609ce1dfc59c/py/configu/commands/eval_command.py#L302) to `:raises: ConfigError`

Reply:
@jamesL92 Thanks ! 🥇 
@rannn505 Thanks, Will change to ConfigError to keep all the errors aligned.